lpd/models.py

```diff
@@ -205,6 +248,35 @@ def type(self):
         """
         return QuestionTypes.MCQ if self.max_options_to_select == 1 else QuestionTypes.MRQ
 
+    @classmethod
+    def _get_score(cls, value):
+        """
+        Return score corresponding to `value`.
+
+        For answer options belonging to multiple choice questions,
+        `value` will be 0 (if learner did not select answer option)
+        or 1 (if learner selected answer option).
+
+        To produce a value that will have the desired effect on recommendations
+        generated by the adaptive engine, we simply need to invert `value`:
+
+        - If a learner selected a given option (i.e., if `value` is 1),
+          this means that the adaptive engine should recommend content associated with that option.
+        - Similarly, if a learner did not select a given option (i.e., if `value` is 1),
```

Comment:
Nit:
```
(i.e., if `value` is 0)
```

Reply:
@tomaszgy Ahhh, this is pretty bad copy-pasta -- thanks for catching it! 😄

Reply:
@tomaszgy Since this is such a small change I'm going to fix-up into 6c5a01b4af77841958dd9bb14d052d943178aece instead of creating a separate commit and squashing later on.